### ISSUE TITLE
copier does not default to copy for local template

### ISSUE THREAD

**Issue by braingram** [copier-org/copier#1234]
### Describe the problem

Based off the documentation:
https://copier.readthedocs.io/en/stable/generating/#regenerating-a-project

I expected that:
```bash
copier template my_project
```

should generate `my_project` based off the template in the `template` directory. However I get an `Unknown sub-command` error:

```
Unknown sub-command 'template'
------
copier 8.0.0

Create a new project from a template.
```
(clipped the remaining help output).

### Template

`template` directory with a single file `copier.yml` with the following contents:
```yaml
my_option:
  type: str
  default: my_value
```

### Expected behavior

Providing the `copy` sub-command works as expected so that `copier copy template my_project` generates:
```
🎤 my_option
   my_value

Copying from template version None
    create  .
```

### Operating system

macOS

### Operating system distribution and version

12.6

### Copier version

copier 8.0.0

### Python version

Python 3.10.6

### Installation method

pip+pypi

**Comment by sisp** [comment]
This is expected as subcommands are required since Copier v8.0.0. Please see the [release notes](https://github.com/copier-org/copier/releases/tag/v8.0.0) for a comprehensive list of breaking (and other) changes.

I'm closing this issue as this change is documented.

**Comment by braingram** [comment]
Thanks for the response.

Than it appears the 'stable' documentation for v8.0.0 is incorrect as
https://copier.readthedocs.io/en/stable/generating/
states:
```
As seen in the quick usage section, you can generate a project from a template using the copier command-line tool:

copier path/to/project/template path/to/destination
```

as does this page:
https://copier.readthedocs.io/en/stable/#quick-start
```
On the command-line:

copier path/to/project/template path/to/destination
```

**Comment by braingram** [comment]
Ah! I see this is fixed in the main branch (so presumably will go out in 8.0.1).

Thanks for the work on this project. I'm just trying to learn it (hence the close attention to the 'quick start' and docs).

**Comment by sisp** [comment]
Ah, that's right, `stable` refers to the latest release, but we overlooked some outdated docs that got corrected by #1192 after that release. Sorry for the inconvenience ... :slightly_frowning_face:

I'm glad to hear you're getting into Copier and going through the docs! :ok_hand: I hope it'll get you started on the fundamentals. Feel free to [ask](https://github.com/copier-org/copier/discussions) in case something isn't clear.
